extra-dev/packages/coq-equations/coq-equations.dev+HoTT/opam

```diff
@@ -0,0 +1,31 @@
+opam-version: "2.0"
+authors: [ "Matthieu Sozeau <matthieu.sozeau@inria.fr>" "Cyprien Mangin <cyprien.mangin@m4x.org>" ]
+maintainer: "matthieu.sozeau@inria.fr"
+homepage: "https://mattam82.github.io/Coq-Equations"
+dev-repo: "git+https://github.com/mattam82/Coq-Equations.git"
+bug-reports: "https://github.com/mattam82/Coq-Equations/issues"
+license: "LGPL 2.1"
+build: [
+  ["./configure.sh HoTT"]
+  [make "-j%{jobs}%"]
+]
+install: [
+  [make "install"]
+]
+remove: ["rm" "-R" "%{lib}%/coq/user-contrib/Equations"]
+depends: [
+  "ocaml"
+  "coq" {= "dev" }
+  "coq-hott" {= "dev" }
+]
+description: """
+Equations is a function definition plugin for Coq, that allows the
+definition of functions by dependent pattern-matching and well-founded,
+mutual or nested structural recursion and compiles them into core
+terms. It automatically derives the clauses equations, the graph of the
+function and its associated elimination principle.
+"""
+synopsis: "A function definition package for Coq+HoTT"
```

Comment:
```suggestion
synopsis: "A function definition package for Coq+HoTT"
tags: [
  "keyword:dependent pattern-matching"
  "keyword:functional elimination"
  "category:Miscellaneous/Coq Extensions"
  "logpath:Equations"
]
```
Suggested metadata from the regular equations package.

Reply:
Thanks